The list of tools, along with the link from which we downloaded the tools, is provided below.

### CAPS [1]
https://caps.tcd.ie/caps/downloads.html
### CoMap [2]
https://jydu.github.io/comap/
### DCA [3]
https://github.com/KIT-MBS/pydca
### GaussDCA [4]
https://github.com/carlobaldassi/GaussDCA.jl
### MIp [5]
https://github.com/diegozea/MIToS.jl
### PSICOV [6]
http://bioinfadmin.cs.ucl.ac.uk/downloads/PSICOV/

### References

1. Fares MA, McNally D. 2006. CAPS: coevolution analysis using protein sequences. Bioinformatics 22:2821-2822.
2. Dutheil J, Galtier N. 2007. Detecting groups of coevolving positions in a molecule: a clustering approach. BMC evolutionary biology 7:1-18.
3. Morcos F, Pagnani A, Lunt B, Bertolino A, Marks DS, Sander C, Zecchina R, Onuchic JN, Hwa T, Weigt M. 2011. Direct-coupling analysis of residue coevolution captures native contacts across many protein families. Proceedings of the National Academy of Sciences 108:E1293-E1301.
4. Baldassi C, Zamparo M, Feinauer C, Procaccini A, Zecchina R, Weigt M, Pagnani A. 2014. Fast and accurate multivariate Gaussian modeling of protein families: predicting residue contacts and protein-interaction partners. PloS one 9:e92721.
5. Dunn SD, Wahl LM, Gloor GB. 2008. Mutual information without the influence of phylogeny or entropy dramatically improves residue contact prediction. Bioinformatics 24:333-340.
6. Jones DT, Buchan DW, Cozzetto D, Pontil M. 2012a. PSICOV: precise structural contact prediction using sparse inverse covariance estimation on large multiple sequence alignments. Bioinformatics 28:184-190.






